components/brave_rewards/resources/page/components/connect_wallet_modal.tsx

```diff
@@ -134,21 +137,35 @@ export function ConnectWalletModal (props: Props) {
                   data-test-id='connect-provider-button'
                   key={provider.type}
                   onClick={onClick}
-                  className={selected ? 'selected' : ''}
+                  className={!provider.enabled ? 'grayed-out' : selected ? 'selected' : ''}
                 >
-                  <style.providerButtonIcon>
-                    {renderProviderIcon(provider.type)}
-                  </style.providerButtonIcon>
-                  <style.providerButtonName>
-                    {provider.name}
-                  </style.providerButtonName>
-                  <style.providerButtonCaret>
-                    <CaretIcon direction='right' />
-                  </style.providerButtonCaret>
+                  <style.grid>
+                    <style.providerButtonIcon>
+                      {renderProviderIcon(provider)}
+                    </style.providerButtonIcon>
+                    <style.providerButtonName>
+                      {provider.name}
+                    </style.providerButtonName>
+                    {!provider.enabled &&
+                      <style.providerButtonMessage>
+                        {getString('connectWalletProviderNotAvailable')}
+                      </style.providerButtonMessage>}
+                    {provider.enabled &&
+                      <style.providerButtonCaret>
+                        <CaretIcon direction='right' />
+                      </style.providerButtonCaret>}
+                  </style.grid>
                 </button>
               )
             })
           }
+          <style.learnMoreLink
+            href={'https://support.brave.com/hc/en-us/articles/6539887971469'}
+            target='_blank'
+            rel='noopener noreferrer'
```

Comment:
There's a component `NewTabLink` in shared that you can use for these kinds of links.